FirebaseCore/Sources/Private/FIRAppInternal.h

```diff
@@ -110,7 +104,7 @@ extern NSString *const FIRAuthStateDidChangeInternalNotificationUIDKey;
  * FIRApp notification listener.
  */
 + (NSError *)errorForSubspecConfigurationFailureWithDomain:(NSString *)domain
```

Comment:
This method is only used once in FDL and once in RC - think it's worth keeping it in Core for a ~5 line implementation? Diff wise it'd probably be less LOC to just inline the implementation at both call sites, and cleans up Core.

Reply:
Done.